client/cli/src/arg_enums.rs

```diff
@@ -59,20 +59,22 @@ pub enum WasmExecutionMethod {
 	Compiled,
 }
 
-impl std::fmt::Display for WasmExecutionMethod {
-	fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-		match self {
-			Self::Interpreted => write!(f, "Interpreted"),
-			Self::Compiled => write!(f, "Compiled"),
+const INTERPRETED_NAME: &str = "interpreted-i-know-what-i-do";
```

Comment:
Any reason why you added this const for the interpreted variant but not for the compiled one?

Reply:
The compiled one follows the name of the enum precisely. The interpreted one has this "interpreted-i-know-what-i-do" extra name. But yeah maybe not necessary.